### PR TITLE
Remove all S commands ##refactoring

### PR DESCRIPTION
**Pull request by Maijin**

### CODE REVIEW

**Comment by radare** [comment]
😱

**Comment by Maijin** [comment]
more like 🤩

**Comment by radare** [comment]
But 😏

**Comment by radare** [comment]
did u took care of this?

* libr/core/project.c:            r_core_cmd (core, "S*", 0);
* libr/core/cmd_anal.c:           r_core_cmdf (core, "S 0x%"PFMT64x" 0x%"PFMT64x" %d %d "

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -3891,8 +3891,6 @@ static void cmd_esil_mem(RCore *core, const char *input) {
 #if 0
```
Comment (radare) [inline libr/core/cmd_anal.c]: delete the whole if 0 block :)

**Comment by radare** [comment]
lgtm, why wan travis red?

**Pull request merged**